docs/_docs/navigation.md

```diff
@@ -353,8 +357,8 @@ In addition to inserting items from the YAML data file into your list, you also
 
 ```liquid
 {% raw %}{% for item in site.data.samplelist.docs %}
-<li class="{% if item.url == page.url %}active{% endif %}">
-<a href="{{ item.url }}">{{ item.title }}</a></li>
+    <li class="{% if item.url == page.url %}active{% endif %}">
+    <a href="{{ item.url }}">{{ item.title }}</a></li>
```

Comment:
- the anchor tag is nested.. so would need to be indented..
- the closing list-item tag on a new-line..